src/charm.py

```diff
@@ -24,11 +29,14 @@
 from ops.model import StatusBase
 
 from config import SparkHistoryServerConfig
-from constants import CONTAINER, INGRESS_REL, PEBBLE_USER, S3_INTEGRATOR_REL
+from constants import CONTAINER, INGRESS_REL, OATHKEEPER_REL, PEBBLE_USER, S3_INTEGRATOR_REL
 from models import S3ConnectionInfo, Status, User
 from utils import WithLogging
 from workload import IOMode, SparkHistoryServer
 
+AUTH_PROXY_ALLOWED_ENDPOINTS = ["version"]
```

Comment:
Does that endpoint actually exist in spark server?

Reply:
Removed, thanks for the suggestion